app/views/ThreadMessagesView/index.tsx

```diff
@@ -43,13 +43,13 @@ import { SubscriptionType, TSubscriptionModel } from '../../definitions/ISubscri
 
 const API_FETCH_COUNT = 50;
 
-interface IResultFetch {
-	threads: IThreadResult[];
-	count: number;
-	offset: number;
-	total: number;
-	success: boolean;
-}
+// interface IResultFetch {
```

Comment:
I didn't want to lose this type. It can be removed when we check this file again.